skyeye-config-example.yaml

```diff
@@ -0,0 +1,10 @@
+log-level: info
+log-format: json
```

Comment:
I only set JSON as default in Windows because pretty gets messed up due to the color codes. Pretty logs are a lot easier to read otherwise. We should probably implement support for https://no-color.org, come to think of it...